src/main/resources/interface-specification.yml

```diff
@@ -10679,30 +10679,25 @@ components:
           format: uuid
       required:
         - id
-    RenewalVerifiedTenantAttributeSeed:
+    UpdateVerifiedTenantAttributeSeed:
       type: object
       properties:
-        renewal:
-          $ref: '#/components/schemas/VerificationRenewal'
         expirationDate:
           type: string
           format: date-time
       required:
-        - renewal
+        - expirationDate
```

Comment:
any reason why this is required?
what should the request be if the use wants to remove the expiration date? or it is not allowed?

Reply:
You're right, I didn't think about if the user wants to remove it, is this allowed? @beetlecrunch 

Now that I think about it, shouldn't either `expirationDate` or `extensionDate` not be an Option in order to verify if the `Attribute` is expired? What happens if both are `None`? Right now we are checking `extensionDate`.

Reply:
Removing expirationDate it's an admitted use case, so it must be optional.